assembl/static2/js/app/utils/translations.js

```diff
@@ -629,6 +629,11 @@ const Translations = {
       down: "Descendre",
       nextStep: "Étape suivante",
       previousStep: "Étape précédente",
+      nameOfTheDebate: "Nom de la discussion",
+      discussionSlug: "Slug de la discussion",
+      slugWarning:
+        "Cette modification entraine un changement de l'adresse de la consultation. Intégrez ce changement dans vos prochaines communications. Pour des assurer la continuité du service, la précédente adresse sera toujours opérationnelle.",
+      invalidSlug: "Les caractères spéciaux ne sont pas autorisés",
```

Comment:
Instead of this message, we should list the allowed characters IMO, `a-z A-Z 0-9 and -`

Reply:
I agree, will talk about it to Fred